docs/tutorials.rst

```diff
@@ -32,70 +33,100 @@ save it as ``rockcraft.yaml``:
+    platforms:
+      amd64:  # Make sure this value matches your computer's architecture
+
+    parts:
       hello:
-          plugin: nil
-          overlay-packages:
-              - hello
+        plugin: nil
+        overlay-packages:
+          - hello
 
-Create OCI image
-----------------
+Pack the ROCK with Rockcraft
+----------------------------
 
-Pack the OCI image, run:
+To build the ROCK, run:
 
 .. code-block:: sh
-		
-  $ rockcraft pack
+
+    $ rockcraft pack
 
 
 The output should look as follows:
 
 .. code-block:: sh
 
-  Launching instance...                                                                                                       
-  Retrieved base ubuntu:20.04                                                                                                 
-  Extracted ubuntu:20.04                                                                                                      
-  Executed: pull hello                                                                                                        
-  Executed: overlay hello                                                                                                     
-  Executed: build hello                                                                                                       
-  Executed: stage hello                                                                                                       
-  Executed: prime hello                                                                                                       
-  Executed parts lifecycle                                                                                                    
-  Created new layer                                                                                                           
-  Cmd set to ['/usr/bin/hello', '-t']                                                                                         
-  Labels and annotations set to ['org.opencontainers.image.version=1.0', 'org.opencontainers.image.title=hello', 'org.opencontainers.image.ref.name=hello', 'org.opencontainers.image.licenses=Apache-2.0', 'org.opencontainers.image.created=2022-06-30T09:07:38.124741+00:00']                                                                                                        
-  Exported to OCI archive 'hello_1.0_amd64.rock'  
+    Launching instance...
+    Retrieved base ubuntu:20.04
+    Extracted ubuntu:20.04
+    Executed: pull hello
+    Executed: overlay hello
+    Executed: build hello
+    Executed: stage hello
+    Executed: prime hello
+    Executed parts lifecycle
+    Created new layer
+    Cmd set to ['/usr/bin/hello', '-t']
+    Labels and annotations set to ['org.opencontainers.image.version=1.0', 'org.opencontainers.image.title=hello', 'org.opencontainers.image.ref.name=hello', 'org.opencontainers.image.licenses=Apache-2.0', 'org.opencontainers.image.created=2022-06-30T09:07:38.124741+00:00']
```

Comment:
TODO: we need to humanize this list (we have a method for this)

Reply:
I assume you'll want to create a separate issue for this. resolving for now, to unblock merge status